server/utils/tans.py

```diff
@@ -1,15 +1,16 @@
 import random
 import string
 
-
-ALLOWED_CHARS = string.ascii_uppercase + string.digits
+import models
+from app import create_app
+from app_config import csrf, db
 
 
-def possible_tans(length: int) -> int:
-    return len(ALLOWED_CHARS) ** length
+ALLOWED_CHARS = string.ascii_uppercase + string.digits
 
 
 class Tan:
+
     value: str = ""
```

Comment:
i know this is not ur code but: what is the purpose of this var?

Reply:
Nothing, actually. Technically not part of this PR, but I'll address that.